kolibri/plugins/coach/assets/src/views/reports/LearnerExerciseDetailPage/index.vue

```diff
@@ -79,7 +95,7 @@
             },
           };
         }
-        return undefined;
+        return {};
```

Comment:
this is result in a different behavior because `undefined` is falsy and an empty object is truthy. was this intentional?

Reply:
Yes, it prevents syntax errors from trying to read properties of undefined.